protocol/test-serialization/src/test_blocks_de.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(test)]
+mod tests {
+    use mina_serialization_types::v1::ExternalTransitionV1;
+    use test_fixtures::TEST_BLOCKS;
+
+    #[test]
+    fn test_deserialize_snark_works_block() {
+        // Grab a block we know to contain snark work from the fixtures
+        let block_fixture = TEST_BLOCKS
+            .get("3NK9fHpzfPWhuxFhQ9Dau1X1JWtstB6kGC4xrurSPU1kctMCsU9U.hex")
+            .unwrap();
+
+        let value: &bin_prot::Value = &block_fixture.value;
+
+        let mut value_bytes = Vec::new();
+        bin_prot::to_writer(&mut value_bytes, value).unwrap();
+
```

Comment:
I think we can delete this file as well as it is covered by the round-trip tests

Reply:
removed

Reply:
removed